Doc/library/stdtypes.rst

```diff
@@ -1369,6 +1369,49 @@ Lists are mutable sequences, typically used to store collections of
 homogeneous items (where the precise degree of similarity will vary by
 application).
 
+.. note::
```

Comment:
I think this needs “Thread safety” in the title.
I think this should work (not tested):
```suggestion
.. admonition:: Thread safety
```

or a custom directive like the existing `ImplementationDetail`